practices/refactoring/readme.md

```diff
@@ -0,0 +1,93 @@
+# Refactoring
+
+Refactoring is very tightly coupled to Simple Design and TDD. TDD enables simple design, while the practice of refactoring ensures that our code bases adapt to new or changing requirements while still keeping its maintainability.
```

Comment:
'is very tightly coupled to' is a bit dramatic, 'plays nicely with', 'relates to', or 'mutually benefits' would probably be more appropriate.

Code that is overly complex and has no tests probably should be refactored to not be overly complex but that doesn't mean anyone followed TDD or that the result will be an exemplar of Simple Design.

Reply:
Simple Design and TDD pages should probably be linked here

Reply:
Got it! Fixed.